core/src/main/java/io/horizontalsystems/core/helpers/HudHelper.kt

```diff
@@ -55,7 +55,7 @@ object HudHelper {
     ): CustomSnackbar? {
 
         val snackbar = CustomSnackbar.make(contenView as ViewGroup, text, backgroundColor, duration, gravity, showProgressBar)
```

Comment:
There is another issue here. `contentView` is `View`, but we expect it to be `ViewGroup`. Should we set its type to `ViewGroup`?